src/main/java/io/appium/java_client/MobileCommand.java

```diff
@@ -282,6 +284,7 @@
         commandRepository.put(COMPARE_IMAGES, postC("/session/:sessionId/appium/compare_images"));
         commandRepository.put(EXECUTE_DRIVER_SCRIPT, postC("/session/:sessionId/appium/execute_driver"));
         commandRepository.put(GET_ALLSESSION, getC("/sessions"));
+        commandRepository.put(EXECUTE_CDP_COMMAND, postC("/session/:sessionId/goog/cdp/execute"));
```

Comment:
the vendor name is configurable. Please rename the constant to EXECUTE_GOOGLE_CDP_COMMAND